src/components/recorder/RightSidePanel.tsx

```
@@ -385,6 +392,12 @@ export const RightSidePanel: React.FC<RightSidePanelProps> = ({ onFinishCapture
           notify('error', t('right_panel.errors.invalid_limit'));
           return;
         }
+
+        const latestListStep = getLatestListStep(browserSteps);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)